.github/workflows/nodejs.yml

```diff
@@ -43,3 +48,8 @@ jobs:
 
       - name: pnpm run postbuild:test
         run: pnpm run postbuild:test
+        # Currently the E2E test is written in bash shell script,
+        # so it is difficult to run on windows. Therefore,
+        # we are temporarily skipping the E2E test on windows.
+        # TODO: Enable E2E test on windows.
+        if: ${{ matrix.os != 'windows-latest' }}
```

Comment:
ref: https://github.com/mizdra/eslint-interactive/issues/251

Reply:
E2E tests could not be run, but I manually checked that it works. eslint-interactive now works on Windows too :tada:

However, there is a bug that the printing of problem details using the pager does not work on windows. I will fix this later.